src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/SyndicationFeed.cs

```diff
@@ -245,8 +245,7 @@ public SyndicationTextInput TextInput
         private SyndicationLink TryReadDocumentationFromExtension(SyndicationElementExtensionCollection elementExtensions)
         {
             SyndicationElementExtension documentationElement = elementExtensions
-                                      .Where(e => e.OuterName == Rss20Constants.DocumentationTag && e.OuterNamespace == Rss20Constants.Rss20Namespace)
-                                      .FirstOrDefault();
+                                      .FirstOrDefault(e => e.OuterName == Rss20Constants.DocumentationTag && e.OuterNamespace == Rss20Constants.Rss20Namespace);
```

Comment:
These aren't tests, they're product src changes.

Reply:
Reverted and moved to a different PR